XCUITests/TopTabsTest.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+let url = "www.mozilla.org"
+let urlLabel = "Internet for people, not profit — Mozilla"
+let urlValue = "mozilla.org"
+
+let urlYah = "www.yahoo.com"
+let urlLabelYah = "Yahoo"
+let urlValueYah = "yahoo"
+
+let urlExample = "example.com"
+
+class TopTabsTest: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testAddTabFromSettings() {
+        navigator.goto(NewTabMenu)
+        app.collectionViews.cells["NewTabMenuItem"].tap()
```

Comment:
Hey @jhugman! One question...I tried to add this to FxScreenGraph, to NewTapMenu scene

> scene: map.createScene(NewTabMenu) { scene in
>         .....
>         scene.tap(app.collectionViews.cells["NewTabMenuItem"], to: NewTabScreen)
>         ......
>     }

But it does work..I also tried with gesture with same result :( It works though if I add that to BrowserTapMenu, and start the test going first to an url...
Don't really understand why it does not work adding it to NewTapMenu...:S 
Any clue? 
If the second solution it's ok for you, I can include it in BrowserTapMenu and do it that way...Thanks!